odxtools/parameters/nrcconstparameter.py

```diff
@@ -88,7 +88,7 @@ def is_required(self) -> bool:
     @property
     @override
     def is_settable(self) -> bool:
-        return False
+        return True
```

Comment:
Sure about this?

Reply:
hm, okay. I've read up the spec about NRC-CONST and I now think that it is supposed to overlapp with a value parameter with the aim of distinguishing negative responses based on the value of the value parameter (i.e., some strange kind of multiplexer?!). when it comes to encoding, it should thus probably not do anything except moving the cursor position. this would require some additional refactoring, mainly in the decoding logic, but I guess I have to think about it a bit more...

Reply:
This is fine from my point of view, just wanted to make sure, since it seemed off topic compared to the rest of the PR

Reply:
Good! I'm constantly surprised about the arcaneness of some of the mechanisms specified by the ODX format. In this case, why are NRC-CONST parameters limited to negative responses (and why are they around in the first place? It seems to me that they can always be replaced by a TABLE-STRUCT parameter)?